### PR TITLE
Change expected account name in NFTs tests to ENS domain

### PR DESCRIPTION
**Pull request by michalinacienciala**
We have configured a ENS name for the
`0x6f1b1f1feb01235e15a7962f16c389c7f8218ed6` address. We need to update the tests to reflect that.

Latest build: [extension-builds-3494](https://github.com/tahowallet/extension/suites/13792525506/artifacts/764721892) (as of Thu, 22 Jun 2023 14:20:28 GMT).

### CODE REVIEW

File: e2e-tests/nfts.spec.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ test.describe("NFTs", () => {
       await page
         .getByTestId("nft_account_filters")
         .filter({
-          hasText: /^(Phoenix|Matilda|Sirius|Topa|Atos|Sport|Lola|Foz)$/,
+          hasText: /^e2e\.testertesting\.eth$/,
```
Comment (jagodarybacka) [inline e2e-tests/nfts.spec.ts:83]: let's add a `const` variable for it so we can reuse that name easily

Comment (jagodarybacka) [reply e2e-tests/nfts.spec.ts]: I would move it to the utils file so every test will have access to that name 😄

Comment (michalinacienciala) [reply e2e-tests/nfts.spec.ts]: [5821c55](https://github.com/tahowallet/extension/pull/3494/commits/5821c559d022ae7265adcbb8458df79c643f9d77)

**Review by jagodarybacka — APPROVED** [review]
✅

**Pull request merged**